docs/userGuide/siteJsonFile.md

```diff
@@ -231,9 +234,9 @@ The example above uses tags as an example of configuring plugin settings, refer
 
 #### **`timeZone`**
 
-**Time zone of the [time stamp](reusingContents.html#built-in-global-variables).** Default: `"UTC"`. 
+**Time zone of the [time stamp](reusingContents.html#built-in-global-variables).** Default: `"UTC"`.
 
-<panel type="minimal" header="Time Zone Options">
+<panel type="seamless" header="Time Zone Options">
```

Comment:
Not too sure if it's possible to add a blank line between
![image](https://user-images.githubusercontent.com/41845017/155429363-5ff6b674-739d-4384-9a9e-c0b3e909e25f.png)

Reply:
I usually end up adding a `<p>` between a panel and the following content.

Reply:
There already is a page break!
<img width="687" alt="Screenshot 2022-02-24 at 2 18 36 PM" src="https://user-images.githubusercontent.com/59471320/155468986-69ffd375-397c-4480-b692-a50560661e85.png">